### DISCUSSION TITLE
Is there a possibility for osu! to come out on other platforms besides PC and mobile?

### DISCUSSION THREAD

**Discussion by RichardKanshen** [ppy/osu#19820] [Ideas]
I don't know development-and-distribution-vise, but it is *technically* possible for the game to be ported onto consoles (? I have no idea).
Consoles already have USB ports, that you can plug-in for example a mouse, keyboard, tablet, other controllers, so it is technically possible for the game to be played on console.

Can I get anybody's opinion on this.

- Richard

**Comment by naipofo** [comment]
Osu-framework is based on SDL2. SDL2 has a lot of target platforms, so it is very much technically possible. I bet one day we will get homebrew ports of lazer to modern consoles, like switch.

**Reply by smoogipoo** [reply]
SDL is the least of concerns. It all depends on whether Mono or .NET Core can run on those platforms. It's also going to require us to get the SDKs.

Put simply, there is a possibility but it's nowhere on the list.

**Comment by dae-vinity** [comment]
> I don't know development-and-distribution-vise, but it is *technically* possible for the game to be ported onto consoles (? I have no idea).
> Consoles already have USB ports, that you can plug-in for example a mouse, keyboard, tablet, other controllers, so it is technically possible for the game to be played on console.
> 
> Can I get anybody's opinion on this.
> 
> - Richard

This is totally possible. Game builder garage has mouse support.
